.circleci/run_docker_build.sh

```diff
@@ -21,6 +20,16 @@ if hash docker-machine 2> /dev/null && docker-machine active > /dev/null; then
     HOST_USER_ID=$(docker-machine ssh $(docker-machine active) id -u)
 fi
 
+if [ -z "${IMAGE_NAME}" ]; then
+    SHYAML_INSTALLED="$(shyaml -h || echo NO)"
```

Comment:
Do we install shyaml anywhere?

Reply:
Yeah it is already being installed in the CI configuration files.